src/scripts/wasting_analyses/analysis_wasting.py

```diff
@@ -0,0 +1,202 @@
+                _col_counter = -1
+            _col_counter += 1  # increment column counter
+            fig.tight_layout()
+        plt.show()
+
+    def plot_wasting_prevalence(self):
+        w_prev_df = self.__logs_dict['wasting_prevalence_count']
+        w_prev_df.set_index(w_prev_df.date.dt.year, inplace=True)
+        w_prev_df.drop(columns='date', inplace=True)
+        w_prev_df = w_prev_df.apply(lambda _row: _row / _row.sum(), axis=1)
+        w_prev_df.plot(kind='bar', stacked=True,
+                       title="Wasting prevalence in children 0-59 months",
+                       ylabel='proportions',
+                       xlabel='year'
+                       )
+        plt.show()
+
+    def plot_modal_gbd_deaths_by_gender(self):
+        """ compare modal and GBD deaths by gender """
+        death_compare = \
+            compare_number_of_deaths(self.__log_file_path, resources)
+        fig, axs = plt.subplots(nrows=1, ncols=2, sharey=True, sharex=True)
+        for _col, sex in enumerate(('M', 'F')):
+            plot_df = death_compare.loc[(['2010-2014', '2015-2019'],
+                                         sex, slice(None), 'Childhood Wasting'
+                                         )].groupby('period').sum()
+            plotting = plot_df.loc[['2010-2014', '2015-2019']]
+            ax = plotting['model'].plot.bar(label='Model', ax=axs[_col], rot=0)
+            ax.errorbar(x=plotting['model'].index, y=plotting.GBD_mean,
+                        yerr=[plotting.GBD_lower, plotting.GBD_upper],
+                        fmt='o', color='#000', label="GBD")
+            ax.set_title(f'{self.__gender_desc[sex]} '
+                         f'wasting deaths, 2010-2014')
+            ax.set_xlabel("Time period")
+            ax.set_ylabel("Number of deaths")
+            ax.legend(loc=2)
+        fig.tight_layout()
+        fig.savefig(
+            outputs / ('modal_gbd_deaths_by_gender' + datestamp + ".pdf"),
+            format="pdf"
+        )
+
+        plt.show()
+
+
+seed = 1
+
+# Path to the resource files used by the disease and intervention methods
+resources = Path("./resources")
+outputs = Path("./outputs")
+
+# create a datestamp
+datestamp = datetime.date.today().strftime("__%Y_%m_%d") + \
+            datetime.datetime.now().strftime("%H_%M_%S")
+
+# configure logging
+log_config = {
+    # output filename. A timestamp will be added to this.
+    "filename": "wasting",
+    "custom_levels": {  # Customise the output of specific loggers
+        "tlo.methods.demography": logging.INFO,
+        "tlo.methods.population": logging.INFO,
+        "tlo.methods.wasting": logging.INFO,
+        '*': logging.WARNING
+    }
+}
+
+# Basic arguments required for the simulation
+start_date = Date(2010, 1, 1)
+end_date = Date(2030, 1, 2)
+pop_size = 10000
+
+# Create simulation instance for this run.
+sim = Simulation(start_date=start_date, seed=seed, log_config=log_config)
+
+# Register modules for simulation
+sim.register(
```

Comment:
did we agree to leave this to @EvaJanouskova when she will be running some analyses?

Reply:
Is it, that I can for simulations use `simplified_births` instead of all the following:
`care_of_women_during_pregnancy`, `labour`, `newborn_outcomes`, `postnatal_supervisor`, and `pregnancy_supervisor`; and use the `full_model` only for checking the calibration?

Reply:
Yes @EvaJanouskova its okay. I guess you will need them for the final calibration(running the full model)

Reply:
@tbhallett, @tamuri, @thewati, @emmanuel, I was also thinking that contraception module may be not necessary for simulation. The only thing I'm still not sure is, how exactly works the approximation when we do not use some of the modules. All the modules related to child mortality, or birth rates will change the number of live births, which will have influence on number of children under 5 and also on number of live births. Does it mean I need these modules as these (nmb of children under 5 & number of live births) are important for me, or is the approximation for all what is covered with the modules (eg antenatal care, increase of contraceptive use in time, ....) well approximated so I can use the model without the modules?